nmpc_settings:
  ros__parameters:
    stateOpti: false
    referencePoseOrTrajectory: 0